src/include/uapi/eq.h

```diff
@@ -36,7 +36,7 @@
 /* Component will reject non-matching configuration. The version number need
  * to be incremented with any ABI changes in function fir_cmd().
  */
-#define SOF_EQ_FIR_ABI_VERSION  1
+#define SOF_EQ_FIR_ABI_VERSION  SOF_ABI_VERSION
```

Comment:
Probably best to just use SOF_ABI_VERSION directly ?

Reply:
This was intended to be extra level of protection agains non-compatible blob for the FIR coefficients format even if the ABI would pass it. If the coefficients format would change in FW an obsolete blob would be rejected. Can the protection be preserved? Fail in this could potentially cause HW damage (speaker burn), fail in standard compliance, excessive sound pressure level, etc.